src/components/proposal/ProposalSummary.tsx

```diff
@@ -105,6 +105,11 @@ function ProposalReview({ confirm }: ProposalSummaryProps) {
           }
         }
       }
+      if (call?.submissionMessage != null) {
+        const defaultMessage =
+          'I am aware that no further edits can be done after proposal submission. \n';
```

Comment:
We wouldn't want the text to always say that no further edits can be done, because that won't always be the case.

I think in thise case you could rename the `checkUpcomingEditableStatus()` function to be something like `updateSubmitButtonMessage()` and have something like:

```javascript
if (proposal.status != null && hasUpcomingEditableStatus) {
  setSubmitButtonMessage(
    'Submit proposal? The proposal can be edited after submission.'.concat(
      call?.submissionMessage ? '\n' + call.submissionMessage : ''
    )
  );
} else {
  setSubmitButtonMessage(
    'I am aware that no further edits can be done after proposal submission.'.concat(
      call?.submissionMessage ? '\n' + call.submissionMessage : ''
    )
  );
}
```

Reply:
Can we not always just show the submission message without any prefix?

Reply:
My apologies, I'm not sure I follow, could you please explain?

Reply:
If I've understood what Fredrik is saying - I think I prefer there being a default message with an additional message below. It means that user officers don't need to remember what the behaviour should be here. It depends on what you were thinking would be useful to add on your side here.

Reply:
I am sorry, I am still not following. Can you provide an example, please?

Reply:
Okay, if you want a default one it is fine. Could we change from "I am aware that no further edits can be done after proposal submission." -> "I am aware that no further edits can be made after proposal submission."